[12.0][BR000275][MIG] Migrate website_header_language_switcher from 8.0 to 12.0 (#246)
Elico-Corp/odoo-8.0: website_header_language_switcher/README.rst

> .. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg

Reviewer: png

   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
   :alt: License: AGPL-3

================================
Website Header Language Switcher
================================

> This module add a language switch in the header of the website.

Reviewer: ```suggestion
This module moves the language switch to the header of the website.
```

Contributors
------------

* Rona Lin <lin.rona@elico-corp.com>

> Maintainer

Reviewer: update to newest version
